app/controllers/coronavirus_form/medical_equipment_type_controller.rb

```diff
@@ -1,50 +1,42 @@
 # frozen_string_literal: true
 
 class CoronavirusForm::MedicalEquipmentTypeController < ApplicationController
-  def show
-    session[:product_details] ||= []
-    @product = find_product(params["product_id"], session[:product_details])
-    super
-  end
-
   def submit
-    session[:product_details] ||= []
-    @product = sanitized_product(params)
+    @form_responses = {
+      medical_equipment_type: strip_tags(params[:medical_equipment_type]).presence,
+    }
 
     invalid_fields = validate_radio_field(
       controller_name,
-      radio: @product[:medical_equipment_type],
+      radio: @form_responses[:medical_equipment_type],
     )
 
     if invalid_fields.any?
       flash.now[:validation] = invalid_fields
       log_validation_error(invalid_fields)
       render controller_path, status: :unprocessable_entity
     elsif selected_testing_equipment?
-      add_product_to_session(@product)
+      update_session_store
       redirect_to testing_equipment_url
     else
-      add_product_to_session(@product)
+      update_session_store
       redirect_to coordination_centres_url
     end
   end
 
 private
 
   def selected_testing_equipment?
-    @product[:medical_equipment_type] == I18n.t(
+    @form_responses[:medical_equipment_type] == I18n.t(
       "coronavirus_form.questions.medical_equipment_type.options.number_testing_equipment.label",
     )
   end
 
-  def previous_path
-    medical_equipment_url
+  def update_session_store
+    session[:medical_equipment_type] = @form_responses[:medical_equipment_type]
```

Comment:
I think this also needs to be moved in the schema too as the session is stored directly to the database with only minor modifications.

At the moment `medical_equipment_type` is only expected to nested inside product_details, and not at the top level of the object. See: https://github.com/alphagov/govuk-coronavirus-business-volunteer-form/blob/ppe_to_regional_centres/config/schemas/form_response.json#L41

I think if left as-is we'll start seeing a lot of errors in Sentry.